packages/paste-core/components/in-page-navigation/stories/index.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import type {Story} from '@storybook/react';
+import {Stack} from '@twilio-paste/stack';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Tabs, TabList, Tab} from '@twilio-paste/tabs';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {useTheme} from '@twilio-paste/theme';
+
+import {InPageNavigation, InPageNavigationItem} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/In Page Navigation',
+  component: InPageNavigation,
+};
+
+export const Default: Story = () => {
```

Comment:
Suggestion (non-blocking): do we need to have a comparison in the story?

Reply:
+1 to removing the comparison

Reply:
I left the comparisons in to make it easier for @twilio-labs/design-systems-pd to review. Maybe this is something we can remove after they've approved it?

Reply:
I think having it initially was good, but post review, let's remove it!